js/scripts/util.js

```diff
@@ -347,9 +347,27 @@ const EVENT_HANDLERS = {
                 break;
         }
     },
-    dialogKeyPress: (key) => {
-        if (key.code === "KeyW") {
-            GAME.user.continuingConversation = true;
+    dialogKeyPress: (e) => {
+        switch (e.code) {
+            case "KeyS":
+                GAME.user.choiceDown = true;
+                break;
+            case "KeyW":
+                GAME.user.choiceUp = true;
+                break;
+        }
+    },
+    // KeyPress does not wanna work for spacebar. Oh well.
```

Comment:
Does jump still work? Does space not work at all or is this a fix?

Reply:
Yes, jump is unchanged. It was a "keypress" vs. "keydown/keyup" error.